lib/crypto/packaged.py

```diff
@@ -86,8 +87,9 @@ def sign_file(file_obj):
         log.info('Not signing file {0}: addon is a hotfix'.format(file_obj.pk))
         return
 
-    # We only sign files that have been reviewed.
-    if file_obj.status not in amo.REVIEWED_STATUSES:
+    # We only sign files that have been reviewed plus beta files.
+    # TODO: Does this make sense? Check repurcussions of this.
+    if file_obj.status not in amo.REVIEWED_STATUSES + (amo.STATUS_BETA,):
```

Comment:
We might want another status for beta files that aren't reviewed yet. We could then simply add `amo.STATUS_BETA` to the list of `amo.REVIEWED_STATUSES`.